plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -896,6 +902,8 @@ func haveSame(a1, a2 []string) bool {
 	return false
 }
 
+// GeneralPredicates includes noncriticalPredicates and EssentialPredicates. noncriticalPredicates are the predicates that
```

Comment:
s/GeneralPredicates includes noncriticalPredicates and EssentialPredicates./GeneralPredicates checks whether noncriticalPredicates and EssentialPredicates pass.